requirements_dev.txt

```diff
@@ -1,12 +1,12 @@
-pytest==3.2.3
-pytest-cov==2.5.1
-coverage==4.0.3
+pytest==5.2.0
+pytest-cov==2.7.1
+coverage==4.5.4
 mock==2.0.0
 
 # PyMongo and Athena dependencies are needed for some of the unit tests:
 # (this is not perfect and we should resolve this in a different way)
 pymongo[tls,srv]==3.6.1
-botocore>=1.12.239,<1.13.0
+botocore==1.12.241
```

Comment:
I thought we wanted to remove this hard pin?

Reply:
Done